py/htm/algorithms/anomaly_likelihood.py

```diff
@@ -613,7 +613,7 @@ def estimateNormal(sampleData, performLowerBoundCheck=True):
 
 
 
-def nullDistribution(verbosity=0):
+def nullDistribution(mean=0.5, verbosity=0):
```

Comment:
I suppose distributionParams is some global/constructor param? Then this could take mean from distrParams as well as the change above?

Reply:
@Zbysekz ^^ otherwise we can merge this

Reply:
Thank you for your work on An Likelihood! Merged.